Minio.Functional.Tests/Program.cs

```diff
@@ -175,7 +175,6 @@ public static void Main(string[] args)
         FunctionalTest.PresignedGetObject_Test3(minioClient).Wait();
         FunctionalTest.PresignedPutObject_Test1(minioClient).Wait();
         FunctionalTest.PresignedPutObject_Test2(minioClient).Wait();
-        FunctionalTest.PresignedGetObject_Test1(minioClient).Wait();
```

Comment:
why is this test removed?

Reply:
A duplicate.
It is also called at the top.